.github/workflows/tests.yml

```diff
@@ -49,6 +49,6 @@ jobs:
         run: make install-nbtest
       - name: Warm up
         continue-on-error: true
-        run: sleep 30 && PATCH_ES=1 ELASTIC_CLOUD_ID=foo ELASTIC_API_KEY=bar bin/nbtest notebooks/search/00-quick-start.ipynb
+        run: sleep 30 && PATCH_ES=1 ELASTIC_CLOUD_ID=foo ELASTIC_API_KEY=bar ES_STACK=${{ matrix.es_stack }} bin/nbtest notebooks/search/00-quick-start.ipynb
```

Comment:
do we need to update the readme to add this env variable?

Reply:
Yes, I'll update the testing instructions in the `CONTRIBUTING.md` file.